messages/es.json

```diff
@@ -5,7 +5,10 @@
   "store/checkout-summary.Shipping": "Entrega",
   "store/checkout-summary.Summary": "Resumen",
   "store/checkout-summary.Total": "Total",
-  "store/checkout-summary.disclaimer": "Gastos de envío e impuestos calculados al finalizar la compra.",
+  "store/checkout-summary.disclaimer": "Gastos de envío e impuestos calculados en el Carrito",
```

Comment:
@camilavcoutinho Can you check these changes?

Reply:
```suggestion
  "store/checkout-summary.disclaimer": "Tasas y fletes calculados en el carrito",
```